src/test/integration.ts

```diff
@@ -69,6 +78,26 @@ describe("integration", () => {
     });
   }
 
+  for (const example of goodWithInfo) {
+
+    const { name, yaml, errPath, rule: exampleRule } = example;
+
+    describe(name, () => {
+      it("should inform with info", () => {
+        const result = defaultLint(yaml);
+        expect(result).to.have.lengthOf(1);
+
+        const [{ rule, type, positions }] = result;
```

Comment:
oh man the double destructure. I didn't even know that was a thing. nice

Reply:
@dexhorthy I rarely use it, only when picking the first element off an array. You can do nested object destructuring, which as you may expect can quickly become horrifying.

See: https://medium.com/@pyrolistical/destructuring-nested-objects-9dabdd01a3b8